packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -7,46 +7,63 @@ import 'package:flutter/foundation.dart';
 import 'deep_links_model.dart';
 import 'fake_data.dart';
 
-
 class DeepLinksController {
-
-  final _linkDatasNotifier = ValueNotifier<List<LinkData>>(allLinkDatas);
   ValueListenable<List<LinkData>> get linkDatasNotifier => _linkDatasNotifier;
+  ValueListenable<String> get searchContentNotifier => _searchContentNotifier;
+  ValueListenable<bool> get showSpitScreenNotifier => _showSpitScreenNotifier;
+
   List<LinkData> get linkDatas => _linkDatasNotifier.value;
 
+  bool get showSpitScreen => _showSpitScreenNotifier.value;
+
+  final selectedLink = ValueNotifier<LinkData?>(null);
+
+  final _linkDatasNotifier = ValueNotifier<List<LinkData>>(allLinkDatas);
   final _searchContentNotifier = ValueNotifier<String>('');
-  ValueListenable<String> get searchContentNotifier => _searchContentNotifier;
-  set searchContent(String content) {
-    _searchContentNotifier.value = content;
-    _updateLinks();
+  final _showSpitScreenNotifier = ValueNotifier<bool>(false);
+
+  set showSpitScreen(bool value) {
+    _showSpitScreenNotifier.value = value;
   }
```

Comment:
If we are going to have a public getter and setter for a notifier, then there is no need to have a private notifier and public ValueListenable. So for this one, I reccomend we just make the `_showSplitScreenNotifier` public